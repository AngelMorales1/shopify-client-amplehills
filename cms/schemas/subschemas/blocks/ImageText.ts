import image from './../image';
import color from './../color';
import portableText from './../portableText';

export default {
  type: 'object',
  name: 'imageText',
  title: 'Image Text',
  fieldsets: [
    {
      name: 'options',
      title: 'Options',
      options: { collapsible: true, collapsed: true }
    },
    {
      name: 'buttons',
      title: 'Buttons',
      options: { collapsible: true, collapsed: true }
    }
  ],
  fields: [
    {
      name: 'title',
      title: 'Title',
      type: 'string'
    },
    {
      name: 'subtitle',
      title: 'Subtitle',
      type: 'string',
    },
    color({
      name: 'subtitleColor',
      title: 'Subtitle Color',
      limit: ['peach', 'blue']
    }),
    portableText({
      name: 'text',
      title: 'Text'
    }),
    image({
      name: 'image',
      title: 'Image'
    }),
    color({
      name: 'backgroundColor',
      title: 'Background Color'
    }),
    {
      name: 'buttonLabel',
      title: 'Button Label',
      fieldset: 'buttons',
      type: 'string'
    },
    color({
      name: 'buttonColor',
      title: 'Button Color',
      fieldset: 'buttons',
      limit: ['peach', 'blue']
    }),
    {
      name: 'buttonLink',
      title: 'Button Link',
      fieldset: 'buttons',
      type: 'url',
      validation: Rule => Rule.uri({ allowRelative: true })
    },
    {
      name: 'textLinkLabel',
      title: 'Text Link Label',
      fieldset: 'buttons',
      type: 'string'
    },
    {
      name: 'textLink',
      title: 'Text Link',
      fieldset: 'buttons',
      type: 'url',
      validation: Rule => Rule.uri({ allowRelative: true })
    },
    {
      name: 'centerAlignTextContent',
      title: 'Center Align Text Content?',
      fieldset: 'options',
      type: 'boolean'
    },
    {
      name: 'drip',
      title: 'Has Drip?',
      fieldset: 'options',
      type: 'boolean'
    },
    {
      name: 'reverse',
      title: 'Reverse Image/Text Arrangement?',
      fieldset: 'options',
      type: 'boolean'
    },
    {
      name: 'fullImage',
      title: 'Scale Image to Fit Box Entirely?',
      fieldset: 'options',
      type: 'boolean'
    },
    {
      name: 'is5050',
      title: 'Is 50% / 50%?',
      fieldset: 'options',
      type: 'boolean',
      initialValue: () => true
    },
  ]
};
