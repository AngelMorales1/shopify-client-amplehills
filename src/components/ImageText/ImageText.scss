@import 'settings', 'typography';

.ImageText {
  @extend .text-madison-blue;

  &--white {
    @extend .bg-white;
  }

  &--pink {
    @extend .bg-light-pink;
  }

  &--blue {
    @extend .bg-sky-blue;
  }

  &--light-peach {
    @extend .bg-light-peach;
  }

  &--peach {
    @extend .bg-peach-pink;
  }

  &--light-pink {
    @extend .bg-baby-pink;
  }

  &--mustard {
    @extend .bg-goldenrod;
  }

  &--yellow {
    @extend .bg-marigold-yellow;
  }

  &--light-yellow {
    @extend .bg-bees-wax;
  }

  &__container {
    flex-direction: column;
    justify-content: space-between;
    margin: 0 spacing(8);

    @media ($medium-up) {
      flex-direction: row;
      margin: 0 spacing(16);
    }

    .ImageText__image {
      min-width: spacing(65);
      margin-top: spacing(12);

      @media ($small-down) {
        transform: translate(0, 0) !important;
      }

      @media ($medium-up) {
        order: 1;
        align-self: flex-end;
        margin-right: spacing(18);
        margin-top: auto;
      }
    }

    .ImageText__text-content {
      @media ($medium-up) {
        order: 2;
        margin-left: spacing(18);
      }
    }
  }

  &__container--reverse {
    .ImageText__image {
      min-width: spacing(65);
      margin-right: auto;

      @media ($medium-up) {
        order: 2;
        align-self: flex-end;
        margin-right: 0;
        margin-left: spacing(18);
      }
    }

    .ImageText__text-content {
      @media ($medium-up) {
        order: 1;
        margin-left: 0;
        margin-right: spacing(18);
      }
    }
  }

  &__container--full-image {
    margin: 0;

    @media ($medium-up) {
      margin: 0;
      flex-direction: row;
      align-items: center;
      justify-content: center;
    }

    .ImageText__text-content {
      padding: 0 spacing(8);

      @media ($medium-up) {
        order: 1;
        margin: 0;
        padding: 0 spacing(16) 0 0;
      }
    }

    .ImageText__text-content--reverse {
      @media ($medium-up) {
        padding: 0 0 0 spacing(16);
      }
    }
  }

  &__button-container {
    justify-content: space-between;

    @media ($medium-up) {
      justify-content: start;
    }
  }

  &__button {
    display: inline-flex;
    padding-top: calc(#{spacing(4)} + 0.125rem);
    padding-bottom: calc(#{spacing(4)} - 0.125rem);
  }

  &__button {
    display: inline-flex;
  }
}

.ImageText__full-image {
  flex-direction: column-reverse;

  @media ($medium-up) {
    flex-direction: row;
  }
}

.ImageText__full-image--reverse {
  @media ($medium-up) {
    flex-direction: row-reverse;
  }
}
