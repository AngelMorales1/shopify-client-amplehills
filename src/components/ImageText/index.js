import React from 'react';
import PropTypes from 'prop-types';
import cx from 'classnames';
import get from 'utils/get';
import imageModel from 'models/imageModel';

import styles from './ImageText.scss';
import { Image } from 'components/base';

<<<<<<< HEAD
const ImageText = ({ block, z }) => {
  const fields = get(block, 'fields', {});
  const colorClass = `ImageText--${get(fields, 'backgroundColor', 'Pink')}`;
  const position = get(fields, 'imagePosition', 0);

  return (
    <div
      style={{ zIndex: z }}
      className={cx('p4 flex drip', styles['ImageText'], styles[colorClass])}
    >
      <div className="flex justify-around px4">
        <Image
          className="z-overlay self-end col-3 square"
          style={{ transform: `translateY(${position}%)` }}
          alt={`${get(fields, 'title', '')} illustration`}
          src={get(fields, 'image.fields.file.url', '')}
        />
        <div className="flex flex-column justify-center my4 col-4">
          <h2 className="block-headline mb3">{get(fields, 'title', '')}</h2>
          <p className="description">{get(fields, 'text', '')}</p>
=======
class ImageText extends Component {
  render() {
    const { block } = this.props;
    const fields = get(block, 'fields', {});
    const colorClass = `ImageText--${get(fields, 'backgroundColor', 'Pink')}`;
    const position = get(fields, 'imagePosition', 0);
    return (
      <div className={cx('flex drip', styles['ImageText'], styles[colorClass])}>
        <div className={cx(styles['ImageText__container'])}>
          <Image
            className={cx(styles['ImageText__image'], 'z-overlay col-4')}
            style={{ transform: `translateY(${position}%)` }}
            alt={`${get(fields, 'title', '')} illustration`}
            src={get(fields, 'image.fields.file.url', '')}
          />
          <div
            className={cx(
              styles['ImageText__text-content'],
              'flex flex-column justify-center'
            )}
          >
            <h2 className="block-headline mb3">{get(fields, 'title', '')}</h2>
            <p className="description">{get(fields, 'text', '')}</p>
          </div>
>>>>>>> mobile style
        </div>
      </div>
    </div>
  );
};

export default ImageText;

ImageText.propTypes = {
  z: PropTypes.number,
  block: PropTypes.shape({
    fields: PropTypes.shape({
      backgroundColor: PropTypes.string,
      image: imageModel.propTypes,
      imagePosition: PropTypes.number,
      text: PropTypes.string,
      title: PropTypes.string
    })
  })
};

ImageText.defaultProps = {
  z: 1,
  block: {
    fields: {
      backgroundColor: 'Pink',
      image: imageModel.default,
      imagePosition: 0,
      text: '',
      title: ''
    }
  }
};
