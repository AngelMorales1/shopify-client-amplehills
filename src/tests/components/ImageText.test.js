import React from 'react';
import { shallow } from 'enzyme';

import ImageText from 'components/ImageText';

it('renders without data', () => {
  const component = shallow(<ImageText />);

  expect(component).toMatchSnapshot();
});

it('renders with data', () => {
  const component = shallow(
    <ImageText
      z={0}
      block={{
        fields: {
          title: 'foo',
          image: {
            sys: {
              space: {
                sys: {
                  type: 'foo',
                  linkType: 'foo',
                  id: 'foo'
                }
              },
              id: 'foo',
              type: 'foo',
              createdAt: 'foo',
              updatedAt: 'foo',
              environment: {
                sys: {
                  id: 'foo',
                  type: 'foo',
                  linkType: 'foo'
                }
              },
              revision: 0,
              locale: 'foo'
            },
            fields: {
              title: 'foo',
              file: {
                url: 'foo',
                details: {
                  size: 0,
                  image: {
                    width: 0,
                    height: 0
                  }
                },
                fileName: 'foo',
                contentType: 'foo'
              }
            }
          },
          text: 'foo',
          imagePosition: 0,
          backgroundColor: 'foo'
        }
      }}
    />
  );

  expect(component).toMatchSnapshot();
});
