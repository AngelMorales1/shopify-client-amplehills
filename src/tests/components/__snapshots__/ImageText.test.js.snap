// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ImageText
    block={
        Object {
            "fields": Object {
              "backgroundColor": "foo",
              "image": Object {
                "fields": Object {
                  "file": Object {
                    "contentType": "foo",
                    "details": Object {
                      "image": Object {
                        "height": 0,
                        "width": 0,
                      },
                      "size": 0,
                    },
                    "fileName": "foo",
                    "url": "foo",
                  },
                  "title": "foo",
                },
                "sys": Object {
                  "createdAt": "foo",
                  "environment": Object {
                    "sys": Object {
                      "id": "foo",
                      "linkType": "foo",
                      "type": "foo",
                    },
                  },
                  "id": "foo",
                  "locale": "foo",
                  "revision": 0,
                  "space": Object {
                    "sys": Object {
                      "id": "foo",
                      "linkType": "foo",
                      "type": "foo",
                    },
                  },
                  "type": "foo",
                  "updatedAt": "foo",
                },
              },
              "imagePosition": 0,
              "text": "foo",
              "title": "foo",
            },
          }
    }
    z={0}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="flex container-width py4"
>
        <div
                className="md-col-6 flex flex-column justify-center col-12"
        >
                <h2
                        className="block-headline mb3"
                >
                        foo
                </h2>
                <p
                        className="block-subheadline"
                        dangerouslySetInnerHTML={
                                Object {
                                        "__html": "<p>foo</p>
                                ",
                                      }
                        }
                />
        </div>
        <Image
                alt="foo illustration"
                bg={false}
                className="z-overlay col-4 mt2"
                onImgLoad={[Function]}
                src="foo?w=1400&fm=png&q=90"
                style={
                        Object {
                                "transform": "translateY(0%)",
                              }
                }
        />
</div>,
      "className": "flex flex-row justify-center drip",
      "style": Object {
        "zIndex": 0,
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="md-col-6 flex flex-column justify-center col-12"
>
            <h2
                        className="block-headline mb3"
            >
                        foo
            </h2>
            <p
                        className="block-subheadline"
                        dangerouslySetInnerHTML={
                                    Object {
                                                "__html": "<p>foo</p>
                                    ",
                                              }
                        }
            />
</div>,
          <Image
            alt="foo illustration"
            bg={false}
            className="z-overlay col-4 mt2"
            onImgLoad={[Function]}
            src="foo?w=1400&fm=png&q=90"
            style={
                        Object {
                                    "transform": "translateY(0%)",
                                  }
            }
/>,
        ],
        "className": "flex container-width py4",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2
                className="block-headline mb3"
>
                foo
</h2>,
              <p
                className="block-subheadline"
                dangerouslySetInnerHTML={
                                Object {
                                                "__html": "<p>foo</p>
                                ",
                                              }
                }
/>,
            ],
            "className": "md-col-6 flex flex-column justify-center col-12",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "foo",
                "className": "block-headline mb3",
              },
              "ref": null,
              "rendered": "foo",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "block-subheadline",
                "dangerouslySetInnerHTML": Object {
                  "__html": "<p>foo</p>
",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "alt": "foo illustration",
            "bg": false,
            "children": null,
            "className": "z-overlay col-4 mt2",
            "onImgLoad": [Function],
            "src": "foo?w=1400&fm=png&q=90",
            "style": Object {
              "transform": "translateY(0%)",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="flex container-width py4"
>
          <div
                    className="md-col-6 flex flex-column justify-center col-12"
          >
                    <h2
                              className="block-headline mb3"
                    >
                              foo
                    </h2>
                    <p
                              className="block-subheadline"
                              dangerouslySetInnerHTML={
                                        Object {
                                                  "__html": "<p>foo</p>
                                        ",
                                                }
                              }
                    />
          </div>
          <Image
                    alt="foo illustration"
                    bg={false}
                    className="z-overlay col-4 mt2"
                    onImgLoad={[Function]}
                    src="foo?w=1400&fm=png&q=90"
                    style={
                              Object {
                                        "transform": "translateY(0%)",
                                      }
                    }
          />
</div>,
        "className": "flex flex-row justify-center drip",
        "style": Object {
          "zIndex": 0,
        },
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="md-col-6 flex flex-column justify-center col-12"
>
              <h2
                            className="block-headline mb3"
              >
                            foo
              </h2>
              <p
                            className="block-subheadline"
                            dangerouslySetInnerHTML={
                                          Object {
                                                        "__html": "<p>foo</p>
                                          ",
                                                      }
                            }
              />
</div>,
            <Image
              alt="foo illustration"
              bg={false}
              className="z-overlay col-4 mt2"
              onImgLoad={[Function]}
              src="foo?w=1400&fm=png&q=90"
              style={
                            Object {
                                          "transform": "translateY(0%)",
                                        }
              }
/>,
          ],
          "className": "flex container-width py4",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2
                  className="block-headline mb3"
>
                  foo
</h2>,
                <p
                  className="block-subheadline"
                  dangerouslySetInnerHTML={
                                    Object {
                                                      "__html": "<p>foo</p>
                                    ",
                                                    }
                  }
/>,
              ],
              "className": "md-col-6 flex flex-column justify-center col-12",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "foo",
                  "className": "block-headline mb3",
                },
                "ref": null,
                "rendered": "foo",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "block-subheadline",
                  "dangerouslySetInnerHTML": Object {
                    "__html": "<p>foo</p>
",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "p",
              },
              null,
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "alt": "foo illustration",
              "bg": false,
              "children": null,
              "className": "z-overlay col-4 mt2",
              "onImgLoad": [Function],
              "src": "foo?w=1400&fm=png&q=90",
              "style": Object {
                "transform": "translateY(0%)",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      null,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="flex container-width py4"
>
            <div
                        className="flex flex-column justify-center col-12 md-col-6"
            >
                        <h2
                                    className="block-headline mb3"
                        >
                                    foo
                        </h2>
                        <p
                                    className="block-subheadline"
                                    dangerouslySetInnerHTML={
                                                Object {
                                                            "__html": "<p>foo</p>
                                                ",
                                                          }
                                    }
                        />
            </div>
            <Image
                        alt="foo illustration"
                        bg={false}
                        className="col-4 z-sub-nav mt2 mx-auto my-auto"
                        onImgLoad={[Function]}
                        src="foo?w=1400&fm=png&q=90"
                        style={
                                    Object {
                                                "transform": "translate(0%, 0%)",
                                              }
                        }
            />
</div>,
          null,
        ],
        "className": "flex justify-center",
        "style": Object {
          "zIndex": 0,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="flex flex-column justify-center col-12 md-col-6"
>
                <h2
                                className="block-headline mb3"
                >
                                foo
                </h2>
                <p
                                className="block-subheadline"
                                dangerouslySetInnerHTML={
                                                Object {
                                                                "__html": "<p>foo</p>
                                                ",
                                                              }
                                }
                />
</div>,
              <Image
                alt="foo illustration"
                bg={false}
                className="col-4 z-sub-nav mt2 mx-auto my-auto"
                onImgLoad={[Function]}
                src="foo?w=1400&fm=png&q=90"
                style={
                                Object {
                                                "transform": "translate(0%, 0%)",
                                              }
                }
/>,
            ],
            "className": "flex container-width py4",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="block-headline mb3"
>
                    foo
</h2>,
                  <p
                    className="block-subheadline"
                    dangerouslySetInnerHTML={
                                        Object {
                                                            "__html": "<p>foo</p>
                                        ",
                                                          }
                    }
/>,
                  null,
                ],
                "className": "flex flex-column justify-center col-12 md-col-6",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "foo",
                    "className": "block-headline mb3",
                  },
                  "ref": null,
                  "rendered": "foo",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "block-subheadline",
                    "dangerouslySetInnerHTML": Object {
                      "__html": "<p>foo</p>
",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "p",
                },
                null,
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "alt": "foo illustration",
                "bg": false,
                "children": null,
                "className": "col-4 z-sub-nav mt2 mx-auto my-auto",
                "onImgLoad": [Function],
                "src": "foo?w=1400&fm=png&q=90",
                "style": Object {
                  "transform": "translate(0%, 0%)",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        null,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`renders without data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ImageText
    block={
        Object {
            "fields": Object {
              "backgroundColor": "pink",
              "buttonColor": "peach",
              "buttonLabel": "",
              "buttonLink": "",
              "drip": false,
              "fullImage": false,
              "image": Object {
                "fields": Object {
                  "file": Object {
                    "url": "",
                  },
                  "title": "",
                },
                "sys": Object {
                  "id": "",
                },
              },
              "imagePosition": 0,
              "isReverseArrangement": false,
              "text": "",
              "title": "",
            },
          }
    }
    z={1}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="flex container-width py4"
>
        <div
                className="md-col-5 flex flex-column justify-center col-12"
        >
                <h2
                        className="block-headline mb3"
                />
                <p
                        className="block-subheadline"
                        dangerouslySetInnerHTML={
                                Object {
                                        "__html": "",
                                      }
                        }
                />
        </div>
        <Image
                alt=" illustration"
                bg={false}
                className="z-overlay col-4 mt2"
                onImgLoad={[Function]}
                src="?w=1400&fm=png&q=90"
                style={
                        Object {
                                "transform": "translateY(0%)",
                              }
                }
        />
</div>,
      "className": "flex flex-row justify-center drip",
      "style": Object {
        "zIndex": 1,
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="md-col-5 flex flex-column justify-center col-12"
>
            <h2
                        className="block-headline mb3"
            />
            <p
                        className="block-subheadline"
                        dangerouslySetInnerHTML={
                                    Object {
                                                "__html": "",
                                              }
                        }
            />
</div>,
          <Image
            alt=" illustration"
            bg={false}
            className="z-overlay col-4 mt2"
            onImgLoad={[Function]}
            src="?w=1400&fm=png&q=90"
            style={
                        Object {
                                    "transform": "translate(0%, 0%)",
                                  }
        "className": "flex container-width py4",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2
                className="block-headline mb3"
/>,
              <p
                className="block-subheadline"
                dangerouslySetInnerHTML={
                                Object {
                                                "__html": "",
                                              }
                }
/>,
            ],
            "className": "md-col-5 flex flex-column justify-center col-12",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "",
                "className": "block-headline mb3",
              },
              "ref": null,
              "rendered": "",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "block-subheadline",
                "dangerouslySetInnerHTML": Object {
                  "__html": "",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "alt": " illustration",
            "bg": false,
            "children": null,
            "className": "z-overlay col-4 mt2",
            "onImgLoad": [Function],
            "src": "?w=1400&fm=png&q=90",
            "style": Object {
              "transform": "translateY(0%)",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="flex container-width py4"
>
          <div
                    className="flex flex-column justify-center col-12 md-col-5"
          >
                    <h2
                              className="block-headline mb3"
                    />
                    <p
                              className="block-subheadline"
                              dangerouslySetInnerHTML={
                                        Object {
                                                  "__html": "",
                                                }
                              }
                    />
          </div>
          <Image
                    alt=" illustration"
                    bg={false}
                    className="z-overlay col-4 mt2"
                    onImgLoad={[Function]}
                    src="?w=1400&fm=png&q=90"
                    style={
                              Object {
                                        "transform": "translateY(0%)",
                                      }
                    }
          />
</div>,
        "className": "flex flex-row justify-center",
        "style": Object {
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="flex flex-column justify-center col-12 md-col-5"
>
              <h2
                            className="block-headline mb3"
              />
              <p
                            className="block-subheadline"
                            dangerouslySetInnerHTML={
                                          Object {
                                                        "__html": "",
                                                      }
                            }
              />
</div>,
            <Image
              alt=" illustration"
              bg={false}
              className="z-overlay col-4 mt2"
              onImgLoad={[Function]}
              src="?w=1400&fm=png&q=90"
              style={
                            Object {
                                          "transform": "translateY(0%)",
                                        }
              }
/>,
          ],
          "className": "flex container-width py4",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2
                  className="block-headline mb3"
/>,
                <p
                  className="block-subheadline"
                  dangerouslySetInnerHTML={
                                    Object {
                                                      "__html": "",
                                                    }
                  }
/>,
                null,
              ],
              "className": "flex flex-column justify-center col-12 md-col-5",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "",
                  "className": "block-headline mb3",
                },
                "ref": null,
                "rendered": "",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "block-subheadline",
                  "dangerouslySetInnerHTML": Object {
                    "__html": "",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "p",
              },
              null,
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "alt": " illustration",
              "bg": false,
              "children": null,
              "className": "z-overlay col-4 mt2",
              "onImgLoad": [Function],
              "src": "?w=1400&fm=png&q=90",
              "style": Object {
                "transform": "translateY(0%)",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      null,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="flex container-width py4"
>
            <div
                        className="flex flex-column justify-center col-12 md-col-5"
            >
                        <h2
                                    className="block-headline mb3"
                        />
                        <p
                                    className="block-subheadline"
                                    dangerouslySetInnerHTML={
                                                Object {
                                                            "__html": "",
                                                          }
                                    }
                        />
            </div>
            <Image
                        alt=" illustration"
                        bg={false}
                        className="col-6 z-sub-nav mt2 mx-auto my-auto"
                        onImgLoad={[Function]}
                        src="?w=1400&fm=png&q=90"
                        style={
                                    Object {
                                                "transform": "translate(0%, 0%)",
                                              }
                        }
            />
</div>,
          null,
        ],
        "className": "flex justify-center",
        "style": Object {
          "zIndex": 1,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="flex flex-column justify-center col-12 md-col-5"
>
                <h2
                                className="block-headline mb3"
                />
                <p
                                className="block-subheadline"
                                dangerouslySetInnerHTML={
                                                Object {
                                                                "__html": "",
                                                              }
                                }
                />
</div>,
              <Image
                alt=" illustration"
                bg={false}
                className="col-6 z-sub-nav mt2 mx-auto my-auto"
                onImgLoad={[Function]}
                src="?w=1400&fm=png&q=90"
                style={
                                Object {
                                                "transform": "translate(0%, 0%)",
                                              }
                }
/>,
            ],
            "className": "flex container-width py4",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="block-headline mb3"
/>,
                  <p
                    className="block-subheadline"
                    dangerouslySetInnerHTML={
                                        Object {
                                                            "__html": "",
                                                          }
                    }
/>,
                  null,
                ],
                "className": "flex flex-column justify-center col-12 md-col-5",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "block-headline mb3",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "block-subheadline",
                    "dangerouslySetInnerHTML": Object {
                      "__html": "",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "p",
                },
                null,
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "alt": " illustration",
                "bg": false,
                "children": null,
                "className": "col-6 z-sub-nav mt2 mx-auto my-auto",
                "onImgLoad": [Function],
                "src": "?w=1400&fm=png&q=90",
                "style": Object {
                  "transform": "translate(0%, 0%)",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        null,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
